src/confluent_kafka/src/confluent_kafka.h

```diff
@@ -42,28 +42,28 @@
  *  0xMMmmRRPP
  *  MM=major, mm=minor, RR=revision, PP=patchlevel (not used)
  */
-#define CFL_VERSION     0x02010000
-#define CFL_VERSION_STR "2.1.0"
+#define CFL_VERSION     0x02010100
+#define CFL_VERSION_STR "2.1.1rc1"
 
 /**
  * Minimum required librdkafka version. This is checked both during
  * build-time (just below) and runtime (see confluent_kafka.c).
  * Make sure to keep the MIN_RD_KAFKA_VERSION, MIN_VER_ERRSTR and #error
  * defines and strings in sync.
  */
-#define MIN_RD_KAFKA_VERSION 0x020100ff
+#define MIN_RD_KAFKA_VERSION 0x020101ff
```

Comment:
MIN_RD_KAFKA_VERSION usually remains the latest minor release, because it's compatible, even if it has bugs. Most times won't matter because same librdkafka version is included in the wheel, but let's follow this rule.